src/lib/components/Modal/ModalHead.scss

```diff
@@ -0,0 +1,38 @@
+@use "sass:map";
+@use "../../styles/tools/spacing";
+@use "settings";
+@use "theme";
+
+.root {
+    display: flex;
+    flex: none;
+    gap: theme.$head-gap;
+    align-items: baseline;
+    padding: settings.$padding-y spacing.of(4) settings.$padding-y settings.$padding-x;
+}
+
+.root h1,
+.root h2,
+.root h3,
+.root h4,
+.root h5,
+.root h6 {
```

Comment:
@adamkudrna I do not like this 💩 code, but I wrote it until you will have a time to look at it. However, I think we need to somehow solve it, otherwise its size and margins would break it